packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -219,13 +226,13 @@ function createProjectionAndParams({
             const apocParamsStr = `{this: ${chainStr || varName}${
                 apocParams.strs.length ? `, ${apocParams.strs.join(", ")}` : ""
             }}`;
-            const apocStr = `${!isPrimitive && !isEnum ? `${param} IN` : ""} apoc.cypher.runFirstColumn("${
+            const apocStr = `${!isPrimitive && !isEnum && !isScalar ? `${param} IN` : ""} apoc.cypher.runFirstColumn("${
```

Comment:
It's getting hard to understand what these checks are checking for. Nothing needs to change now but either a comment or refactor the check into a sensibly named fn might be something to consider for the future.